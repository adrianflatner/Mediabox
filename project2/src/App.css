.Wrapper {
  text-align: center;
  display:grid;
  grid-template-columns: 1fr;
  justify-items: center;
  grid-auto-rows: minmax(30px, auto);
}


.Title{
  color:#707070;
  grid-row: 2/3;
}

.Navbar{
  grid-row:4/5;
}

.Artboard{
  background-color: #ED553B;
  grid-row:5/6;
}

.Artboard-tools{
  background-color:#20639B;
  grid-row:6/7;
  width:70%;
}


.Wrapper{
  
}


#placeholder{
  max-width: 960px;
}

body{
  background-image: linear-gradient(120deg, #a6c0fe 0%, #f68084 100%);

}
