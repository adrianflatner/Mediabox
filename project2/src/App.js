import React from 'react';
import './App.css';
import Tools from './components/Tools.js';
import placeholder from './components/pictures/placeholder.jpg';
import './App.css';
import Navbar from './components/Navbar.js';

function App() {
  return (
    <div className="Wrapper">
      
        <div className="Title">
            <h1>I really like your shirt, homo</h1>
            <p>thanks dad</p> 
        </div>

        <div className="Navbar"><Navbar /></div>
        <div className="Artboard"><img id="placeholder" src={placeholder} alt="placeholder"/></div>
        <div className="Artboard-tools"><Tools /></div>


     

      {/* <header className="App-header">
        <img src={logo} className="App-logo" alt="logo" />
        <p>
          Edit <code>src/App.js</code> and save to reload.
        </p>
        <a
          className="App-link"
          href="https://reactjs.org"
          target="_blank"
          rel="noopener noreferrer"
        >
          Learn React
        </a>
      </header> */}
    </div>
  );
}

export default App;
