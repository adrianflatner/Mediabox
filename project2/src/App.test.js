import React from 'react';
import ReactDOM from 'react-dom';
import App from './App';
import renderer from 'react-test-renderer';

it('renders without crashing', () => {
	const div = document.createElement('div');
	ReactDOM.render(<App />, div);
	ReactDOM.unmountComponentAtNode(div);
});

describe('App', () => {
	test('loads without error', () => {
		let element = <App />;

		expect(element).toBeTruthy();
	});
});

describe('App', () => {
	test('render matches snapshot', () => {
		const component = renderer.create(<App />);
		const tree = component.toJSON();

		expect(tree).toMatchSnapshot();
	});
});