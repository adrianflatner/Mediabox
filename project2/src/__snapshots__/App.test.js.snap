// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App render matches snapshot 1`] = `
<div
  className="Wrapper"
>
  <div
    className="Title"
  >
    <h1>
      Welcome to our Artboard Machine
    </h1>
  </div>
  <div
    className="Navbar"
  >
    <div>
      <button
        className="Save"
        onClick={[Function]}
      >
        save current combination
      </button>
      <button
        className="Load"
        onClick={[Function]}
      >
        Load saved combination
      </button>
      <div
        className="grid"
      >
        <button
          className="tab1"
          onClick={[Function]}
        >
          Trump
        </button>
        <button
          className="tab2"
          onClick={[Function]}
        >
          Richard
        </button>
        <button
          className="tab3"
          onClick={[Function]}
        >
          Jaden
        </button>
        <button
          className="tab4"
          onClick={[Function]}
        >
          Michael
        </button>
      </div>
      <div
        className="Artboard-tools"
      >
        <div
          className="Wrap"
        >
          <div
            className="Artboard"
          >
            <div>
              <div
                dangerouslySetInnerHTML={
                  Object {
                    "__html": "",
                  }
                }
              />
              <div
                id="quote"
              >
                "
                
                "
              </div>
              <audio
                controls={true}
                loop={true}
              >
                <source
                  src=""
                  type="audio/mpeg"
                />
                Your browser does not support the audio element.
              </audio>
            </div>
          </div>
          <div
            className="textboxes"
            id="pictures"
          >
            <h2
              class="h2"
            >
              Photos:
            </h2>
            <label
              class="container"
            >
              Hair
              <input
                className="checkbox"
                name="group 1"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Retrd
              <input
                className="checkbox"
                name="group 1"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Smile
              <input
                className="checkbox"
                name="group 1"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
          </div>
          <div
            className="textboxes"
            id="sounds"
          >
            <h2
              class="h2"
            >
              Sounds:
            </h2>
            <label
              class="container"
            >
              Ueeaa
              <input
                className="checkbox"
                name="group 2"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Grab pussy
              <input
                className="checkbox"
                name="group 2"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                className="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Church
              <input
                className="checkbox"
                name="group 2"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
          </div>
          <div
            className="textboxes"
            id="quotes"
          >
            <h2
              class="h2"
            >
              Quotes:
            </h2>
            <label
              class="container"
            >
              Joakim
              <input
                className="checkbox"
                name="group 3"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Halvor
              <input
                className="checkbox"
                name="group 3"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
            <br />
            <label
              class="container"
            >
              Adrian
              <input
                className="checkbox"
                name="group 3"
                onChange={[Function]}
                type="checkbox"
              />
              <span
                class="checkmark"
              />
            </label>
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
